app/components/long_reads/builder.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module LongReads
+  # A builder for articles that come from Long Reads
+  class Builder < GenericBuilder
+    private
+
+    def publication
```

Comment:
Doesn't depend on instance state (maybe move it to another class?) (UtilityFunction)